explorer/src/providers/cluster.tsx

```diff
@@ -67,11 +67,25 @@ export function clusterUrl(cluster: Cluster, customUrl: string): string {
   }
 }
 
+export function solarweaveUrl(cluster: Cluster, solarweaveUrl: string): string {
+  switch (cluster) {
+    case Cluster.Devnet:
+      return "solana-devnet-solarweave";
+    case Cluster.MainnetBeta:
+      return "solana-mainnet-solarweave";
+    case Cluster.Testnet:
+      return "solana-testnet-solarweave";
```

Comment:
Do these already have data or are they placeholders?

Reply:
Only devnet is partially populated.
These are currently placeholders.

I still need archive nodes that have block data from block 0 for each network.